mmdet/models/roi_heads/mask_heads/fused_semantic_head.py

```diff
@@ -4,6 +4,7 @@
 from mmcv.runner import auto_fp16, force_fp32
 
 from mmdet.models.builder import HEADS
+from mmdet.models.utils import ResLayer, SimplifiedBasicBlock
 
 
 @HEADS.register_module()
```

Comment:
Similarly, we think we may keep `fused_semantic_head.py` unchanged.
Then, we could add a new mask head for the desired function.